### PR TITLE
Feature/vot 93 landing page with must haves

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Short description
This PR changes the Landing Page's design to the Aragon look and feel design. 

## How can it be tested
Open the staging version of the website and check that the design 

- is aligned with the design on Figma
- satisfies all criteria specified by the corresponding issue

## Tracker ID
https://linear.app/aragon/issue/VOT-93/landing-page-with-must-haves

### CODE REVIEW

File: components/layout.tsx [hunk 1]
```diff
@@ -5,15 +5,16 @@ import { Header } from "./header";
...
   padding: 0 ${({ theme }) => theme.margins.desktop.horizontal};
   margin-top: 77px;
-  max-width: 992px;
+  max-width: 1249px;
   margin-left: auto;
   margin-right: auto;
-
-  @media ${({ theme }) => theme.screens.tablet} {
+  
+  @media ${size.tablet} {
```
Comment (cbrzn) [inline components/layout.tsx:17]: Let's import this from the theme directly, instead of the theme file

File: components/token-card.tsx [hunk 2]
```diff
@@ -4,30 +4,31 @@ import Link from "next/link";
...
   &:hover {
-    box-shadow: 1px 4px 9px rgba(0, 0, 0, 0.35);
-    opacity: 0.95;
+    box-shadow: 0px 6px 6px rgba(180, 193, 228, 0.35);
+    background: linear-gradient(
+      101.6deg,
+      rgb(102,218,255,0.2),
+      rgb(1, 232, 247,0.2),
+      rgb(128, 247, 255,0.2)
+    );
```
Comment (cbrzn) [inline components/token-card.tsx:24]: IIRC we should import these colors from the theme

Comment (WissenIstNacht) [reply components/token-card.tsx:24]: I think this was pulled in from Nestor (I don't recall working on this). Are you sure there was not a reason for leaving it like this?

Comment (cbrzn) [reply components/token-card.tsx:24]: Ohhh okok fresh - Yah I think this was left behind :P, we should change this

Good job so far bro

Comment (nestorbe) [reply components/token-card.tsx:24]: Yes, that was me :) changing it right now!

**Review by cbrzn — APPROVED** [review]

**Review by nestorbe — APPROVED** [review]

**Pull request merged**